docs/_basic/listening_events.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Listening to events
+lang: en
+slug: event-listening
+order: 3
+---
+
+<div class="section-content">
+
+You can listen to [any Events API event](https://api.slack.com/events) using the `event()` method after subscribing to it in your app configuration. This allows your app to take action when something happens in Slack, like a user reacting to a message or joining a channel.
+
+The `event()` method requires an `eventType` of type `str`.
+
+</div>
+
+```python
+# When a user joins the team, send a message in a predefined channel asking them to introduce themselves
+@app.event("team_join")
+def ask_for_introduction(payload, say):
+    welcome_channel_id = "C12345";
+    user_id = payload["event"]["user"]
+    text = f"Welcome to the team, <@{user_id}>! 🎉 You can introduce yourself in this channel."
+    say(text=text, channel=welcome_channel_id)
```

Comment:
As we discussed, `payload` is no longer equivalent to `body`. The current code doesn't work (this is the same for other samples using `paylaod`)
```suggestion
@app.event("team_join")
def ask_for_introduction(event, say):
    welcome_channel_id = "C12345";
    user_id = event["user"]["id"]
    text = f"Welcome to the team, <@{user_id}>! 🎉 You can introduce yourself in this channel."
    say(text=text, channel=welcome_channel_id)
```